app/models/manual_record.rb

```diff
@@ -76,7 +76,7 @@ class Edition
     field :state, type: String
     field :version_number, type: Integer
     field :document_ids, type: Array
-    field :removed_document_ids, type: Array
+    field :removed_document_ids, type: Array, as: :removed_section_ids
```

Comment:
I tried to find the documentation for the `as` option just in case it changes/goes away when #929 lands. I don't think we need to worry about it as this change goes away from 335688 onwards.